### PR TITLE
Curation viz

### PR DESCRIPTION
**Pull request by FlynnOConnell**

### CODE REVIEW

File: masknmf/demixing/demixing_results.py [hunk 1]
```diff
@@ -385,7 +385,8 @@ def _move_managed_tensors(self, new_device: str):
     def _move_managed_arrays(self, new_device: str):
         for arr_name in self._managed_arrays:
             curr_arr = getattr(self, arr_name)
-            curr_arr.to(self.device)
```
Comment (apasarkar) [inline masknmf/demixing/demixing_results.py:388]: @FlynnOConnell did you find cases where the managed arrays were None?

File: pyproject.toml.template [hunk 2]
```diff
@@ -0,0 +1,18 @@
+[project]
```
Comment (apasarkar) [inline pyproject.toml.template]: @FlynnOConnell what's the purpose of this file?

Comment (FlynnOConnell) [reply pyproject.toml.template]: I forgot about this file, i use it for the [torch.tool] entries which handle the torch cuda version for me, can remove or keep

File: masknmf/demixing/demixing_results.py (hunk 1 above)
Comment (FlynnOConnell) [reply masknmf/demixing/demixing_results.py:388]: Only if there are no correlation images in the demixing results, e.g. `standard_correlation_images` or `residual_correlation_images` weren't ran, which shouldn't happen (?)

File: pyproject.toml.template (hunk 2 above)
Comment (apasarkar) [reply pyproject.toml.template]: @FlynnOConnell pls remove!

File: notebooks/classification_labeling.ipynb [hunk 3]
```diff
@@ -0,0 +1,151 @@
+{
+ "cells": [
+  {
```
Comment (apasarkar) [inline notebooks/classification_labeling.ipynb]: @FlynnOConnell can you remove this and share directly with me? I'll add a demo

File: masknmf/visualization/imgui/movie_player.py [hunk 4]
```diff
@@ -0,0 +1,94 @@
+import time
+from typing import *
+import numpy as np
+from imgui_bundle import imgui
+
+
+class MoviePlayer:
+    """
```
Comment (apasarkar) [inline masknmf/visualization/imgui/movie_player.py:8]: @FlynnOConnell can you add intended use documentation here (like you did with the Checkbox functionality) so it's clear how to use this building block with fastplotlib figures?

File: masknmf/demixing/demixing_results.py (hunk 1 above)
Comment (apasarkar) [reply masknmf/demixing/demixing_results.py:388]: @FlynnOConnell just checking tho - when you ran this, did the code ever actually crash?

Comment (FlynnOConnell) [reply masknmf/demixing/demixing_results.py:388]: It crashed when calling `from_hdf5()` on a demixing results that didn't have standard correlation images, though this was probably due to a failed run, can revert this if you want becaue that won't happen under a standard pipeline run, I think mine failed but i don't remember

File: masknmf/multisession/roicat_tracking.py [hunk 5]
```diff
@@ -67,6 +67,29 @@ def session_files(self) -> tuple[str | tuple[str, ...]]:
         """
         return self._session_files
 
+    def set_class_labels(self, labels=None, path_labels=None, n_classes=None) -> None:
```
Comment (apasarkar) [inline masknmf/multisession/roicat_tracking.py]: The labels property in the roicat_classifier already had a check in the setter that all the lengths matched. 

I think it makes sense for the rest of this logic to go there. What do you think?

File: masknmf/classification/roicat_classification.py [hunk 6]
```diff
@@ -1,87 +1,145 @@
...
+    CLASSIFIER_SUFFIX,
+    read_labels,
+    write_labels,
+    write_predictions,
+)
+from masknmf.multisession.roicat_tracking import RoicatDataAdapter
 from masknmf.utils import torch_select_device, display
-import tempfile
+
+TRAINING_SUFFIX = ".training.json"
```
Comment (apasarkar) [inline masknmf/classification/roicat_classification.py:23]: @FlynnOConnell what's the purpose of this one + lines below it?

**Review by apasarkar — CHANGES_REQUESTED** [review]

**Review by apasarkar — CHANGES_REQUESTED** [review]
soefjao

File: masknmf/classification/roicat_classification.py (hunk 6 above)
Comment (FlynnOConnell) [reply masknmf/classification/roicat_classification.py:23]: This is the filename of the class labels and label names
